docker/addons/cassandra/init.sh

```diff
@@ -0,0 +1,3 @@
+docker-compose -f docker/docker-compose.yml -f docker/addons/cassandra/docker-compose.yml up -d
```

Comment:
What will happen if its running already and I run `./docker/addons/cassandra/init.sh` again as README says ?